packages/pds/src/api/app/bsky/getUsersTypeahead.ts

```diff
@@ -1,4 +1,4 @@
-import { QueryParams } from '@atproto/api/src/types/app/bsky/getUsersTypeahead'
+import { AppBskyGetUsersTypeahead as Method } from '@atproto/api'
```

Comment:
we should import these from the server types (pds/src/lexicon) instead of the api package